spot.md

```diff
@@ -112,13 +112,17 @@ Because data isn't explicitely initialized to 0 in KMCD, we need explicit initia
 ```k
     syntax SpotAuthStep ::= "init"     String
                           | "setPrice" String Wad
+                          | "setMat"   String Ray
```

Comment:
Why not `"file" String String Ray` with the appropriate reversion behavior to mirror the contract implementation?

Reply:
https://github.com/makerdao/dss/blob/master/src/spot.sol#L87

Reply:
good catch